bundle/jsky.app.ot/src/main/scala/jsky/app/ot/ags/BagsManager.scala

```diff
@@ -32,8 +33,18 @@ import Scalaz._
 import scalaz.effect.IO
 import scalaz.effect.IO.ioUnit
 
+/** The status of BAGS for a given observation, for UI feedback purposes. */
+sealed class BagsStatus
+
+object BagsStatus {
+  case object ErrorStatus                extends BagsStatus
+  case object PendingStatus              extends BagsStatus
+  case object RunningStatus              extends BagsStatus
+  case class  FailureStatus(why: String) extends BagsStatus
+}
+
```

Comment:
Could we make the transition methods `private[BagsManager]` or something if we're concerned about that? As you say it's just a potential confusion, not dangerous.